### ISSUE TITLE
More special dialogue when you put the character files back in the folder?

### ISSUE THREAD

**Issue by steve02081504** [Monika-After-Story/MonikaModDev#10141]
I tested it and it looks like MAS is not responding in the following places where users expect special dialogue:

When monika is indoors:
- The user places the original DDLC's `monika.chr` file to the `characters` folder
- User places the original DDLC `natsuki.chr` file into the `characters` folder.
- User placed the original DDLC's `sayori.chr` file into the `characters` folder.
- User places the original DDLC's `yuri.chr` file into the `characters` folder.
- User places another backed up MAS version of `monika` file into `characters` folder

When monika is not in the room (taken out):
- User places the original DDLC `monika.chr` file into the `characters` folder.
- User places the original DDLC `natsuki.chr` file into the `characters` folder.
- User placed the original DDLC's `sayori.chr` file into the `characters` folder.
- User places the original DDLC `yuri.chr` file into the `characters` folder.

**Comment by dreamscached** [comment]
As a result of #9641 `.chr` files are no longer by MAS and are automatically removed on startup. They do not have any meaning to MAS anymore, they were used in DDLC.

**Comment by steve02081504** [comment]
> https://github.com/Monika-After-Story/MonikaModDev/issues/10141#issuecomment-1807128705

You're right, but I as a user put the `.chr` file back into the `characters` folder just to see if there's any special dialogue and expect some cool hidden scripts, and monika's inaction on this is a bit immersion breaking for me
So I think MAS might be able to add some dialogue in that regard, that's all.

**Comment by Booplicate** [comment]
Monika explains that the files have no special meaning and the ndeletes them. What reaction would you expect?

**Comment by steve02081504** [comment]
> https://github.com/Monika-After-Story/MonikaModDev/issues/10141#issuecomment-1807144341

I think the player trying to put `.chr` files back could be seen as an attempt to bring back the character in the game. monika can tease on this and then deleting the file again, isn't she?
Backing up a bit, if we really think the `.chr` file doesn't mean anything and don't add anything to it
- When the player chose to take monika out of the room, he made multiple copies of the `monika` file generated by MAS.
- then brings back a copy of the `monika` file to the `characters` folder when monika is in the room afterwards.

This is a situation that I think monika should have reacted to.

**Comment by Booplicate** [comment]
> monika can tease on this and then deleting the file again, isn't she?

So teasing, that might work, but would require some brainstorming.

> This is a situation that I think monika should have reacted to.

For `monika` file see #5026, there's definitely will be a reaction, but it's tricky to do right because of how many paths there are.
